resources/fuzzing_presentations.md

```diff
@@ -0,0 +1,10 @@
+**Presentations**
```

Comment:
I think we should move this one in program-analysis/echidna, and mention the page in Echidna's readme